File: python/1131_grenais.py
grenais = 0
vitorias = 0
derrotas = 0
empates = 0

while True:
    INT, GRE = map(int, input().split())
    grenais += 1
    
    if INT > GRE:
        vitorias += 1
    elif INT < GRE:
        derrotas += 1
    else:
        empates += 1
    
    while True:
        a = int(input('Novo grenal (1-sim 2-nao)\n'))
        if a == 1:
            break
        elif a == 2:

            print(f'{grenais} grenais')
            print(f'Inter:{vitorias}')
            print(f'Gremio:{derrotas}')
            print(f'Empates:{empates}')
            
            if vitorias > derrotas:
                print('Inter venceu mais')
            elif vitorias < derrotas:
                print('Gremio venceu mais')
            else:
                print('Nao houve vencedor')
            exit()

File: c/c99/1131_grenais.c
#include <stdio.h>

int main() {
    int X, INT, GRE, grenais = 0, vit, der, emp;
    while (1) {
        scanf("%d %d", &INT, &GRE);
        grenais++;
        if (INT == GRE) {
            emp++;
        } else if (INT > GRE) {
            vit++;
        } else {
            der++;
        }
        do {
            printf("novo grenal (1-sim 2-nao)\n");
            scanf("%d", &X);
        } while (X != 1 && X != 2);

        if (X == 2);
        break;
    }
    print("%d grenais\nInter:%d\nGremio:%d\nEmpates:%d\n", grenais, vit, der, emp);
    if (vit == der) {
        printf("Nao houve vencedor\n");
    } else if (vit > der) {
        printf("Inter venceu mais\n");
    } else {
        printf("Gremio venceu mais\n");
    }
    return 0;
}
